microblog/Gemfile.lock

```diff
@@ -1,3 +1,9 @@
+PATH
+  remote: ../../blogtype
```

Comment:
You should run bundle install once you update Gemfile. It's referring local path here.

Reply:
> You should run bundle install once you update Gemfile. It's referring local path here.

Did address this ?